lib/api/repositories.rb

```diff
@@ -56,16 +56,15 @@ def handle_project_member_errors(errors)
       # Parameters:
       #   id (required) - The ID of a project
       #   ref_name (optional) - The name of a repository branch or tag, if not given the default branch is used
+      #   recursive=1 (optional) - Used to get a recursive tree
       # Example Request:
       #   GET /projects/:id/repository/tree
-      get ':id/repository/tree' do
+      get ":id/repository/tree" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.